### PR TITLE
#465  Render tags on product page

### LINKED ISSUE #403: Resolve segfault issue at CI tests
This issue also will resolve this not auto created pdd issue:

Resurrect product.diameter_out field render
https://github.com/fidals/stroyprombeton/pull/391/files#diff-0935e97978f420ce3f93d59b1560e197R62

### LINKED ISSUE #465: Render tags on product page
~Connect tags to options instead of products. Create fixtures for it.~
Fixtures already created. So, i'll use it to render tags on product page.

### PR DESCRIPTION
**Pull request by duker33**
Closes #465 
Closes #403

### CODE REVIEW

File: stroyprombeton/views/catalog.py [hunk 1]
```diff
@@ -135,11 +135,17 @@ def get_context_data(self, **kwargs):
             {'name': name, 'url': url()}
             for (name, url) in product_ancestors[offset:offset + limit]
         ]
+        tag_groups = (
+            models.Tag.objects
+            .filter_by_products([product])
```
Comment (artemiy312) [inline stroyprombeton/views/catalog.py]: Let's create a todo to implement `filter_by_options` and drop filtering of option's tags  in `product.html`

File: stroyprombeton/tests/tests_views.py [hunk 2]
```diff
@@ -381,6 +339,24 @@ def test_inactive_product_unavailable(self):
         self.assertEqual(response.status_code, 404)
         ModelPage.objects.filter(stroyprombeton_product=product_id).update(is_active=True)
 
+    def test_tags_table(self):
+        """Options table should contain right tags set."""
+        product_id = 26
```
[the remark this answers is not in the document]
Comment (duker33) [reply stroyprombeton/tests/tests_views.py]: @artemiy312 , i'll query product with a name or with id. It's both fragile. But i'll change it to name, no problem

[the remark this answers is not in the document]
Comment (artemiy312) [reply stroyprombeton/tests/tests_views.py]: @duker33 I meant we should get a `Product` instance without any hard-coded data, that may be changed

Example:
```
Product.objects.filter(tags__isnull=False).first()
```

**Comment by duker33** [comment]
merge it now. Business hardly waits stage with this code

**Pull request merged**